src/service/localization/strings.js

```diff
@@ -111,6 +111,9 @@ export const LocalizedStringId_Enum = {
   AMP_STORY_INTERACTIVE_QUIZ_ANSWER_CHOICE_C: '73',
   AMP_STORY_INTERACTIVE_QUIZ_ANSWER_CHOICE_D: '74',
 
+  // amp-story-shopping
+  AMP_STORY_SHOPPING_SHOP_LABEL: '101',
```

Comment:
nit: 'CTA' in the name could help describe its usage.  
This could be changed to `AMP_STORY_SHOPPING_CTA_LABEL`.